scripts/dashboard/configure_kibana_dashboard.sh

```diff
@@ -1,10 +1,5 @@
 #!/bin/bash
 
-curl -X PUT -H "kbn-version: 5.5.2" -d '{"title":"wikipediabot","notExpandable":true, "timeFieldName": "META.DT"}' http://localhost:5601/es_admin/.kibana/index-pattern/wikipediabot/_create
-curl -X POST -H "kbn-version: 5.5.2" -H "Content-Type: application/json;charset=UTF-8" -d '{"value":"wikipediabot"}' http://localhost:5601/api/kibana/settings/timelion:es.default_index
-curl -X POST -H "kbn-version: 5.5.2" -H "Content-Type: application/json;charset=UTF-8" -d '{"value":"META.DT"}' http://localhost:5601/api/kibana/settings/timelion:es.timefield
-curl -X POST -H "kbn-version: 5.5.2" -H "Content-Type: application/json;charset=UTF-8" -d '{"value":"wikipediabot"}' http://localhost:5601/api/kibana/settings/defaultIndex
-curl -X PUT -H "kbn-version: 5.5.2" -d '{"title":"en_wikipedia_gt_1","notExpandable":true}' http://localhost:5601/es_admin/.kibana/index-pattern/en_wikipedia_gt_1/_create 
-curl --silent --output /dev/null -X POST -H 'kbn-xsrf:true' -H 'Content-type:application/json' -d @scripts/dashboard/kibana_dash.json http://localhost:5601/api/kibana/dashboards/import
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
-#curl -X GET http://localhost:5601/api/kibana/dashboards/export?dashboard=Wikipedia > /tmp/dash.json
+curl -XPOST "http://localhost:5601/api/saved_objects/_import?overwrite=true" -H "kbn-xsrf: true" -H "securitytenant: global" --form file=@${DIR}/cp-demo.ndjson
```

Comment:
I'm sure this works, but I'm used to seeing a space here and for readability `-X POST` - is nospace better?

Reply:
PR is updated with space